### PR TITLE
Task for max_attendees to have a value when require_registration is set

### PR DESCRIPTION
**Pull request by differentreality**
Fixes https://github.com/openSUSE/osem/issues/991
`bundle exec rake events_registrations:set_max_attendees`

### CODE REVIEW

**Comment by differentreality** [comment]
Note: Specially for oSC16, the submitter of the event must go and edit the proposal to add the actual max_attendees value.

File: lib/tasks/events_registrations.rake [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+  task set_max_attendees: :environment do
+    @events = Event.all.where(require_registration: true, max_attendees: nil)
+    puts "Fixing max_attendees attribute for #{@events.length} events."
+    puts "The IDs of those events are: #{@events.pluck(:id)}"
+
+    @events.each do |event|
+      if event.registrations.any?
+        event.max_attendees = event.registrations.count
+      else
+        event.max_attendees = 1
```
Comment (hennevogel) [inline lib/tasks/events_registrations.rake]: Erm this is not really what we want right?

Comment (differentreality) [inline lib/tasks/events_registrations.rake]: Not really, but I am not sure what the alternative is.
We have no way of knowing the value of max_attendees. OK it is probably not going to be 1, but we don't know if it is 10 or 30 or...

Comment (hennevogel) [inline lib/tasks/events_registrations.rake]: Then I don't think we should rely on it

**Comment by hennevogel** [comment]
:-1:  I think instead of adding a rather random number to `max_attendees` we should get rid of the `max_attendees_and_require_registration` validation because it really should be possible to require registration for the Event without having a limit

**Pull request closed without merging**